source/includes/markdown/_07-auth.md

```diff
@@ -229,6 +239,50 @@ To implement the Authorization Code Grant flow (the most typical flow for most a
 
 The token that you have at the end can be used to make calls to the Asana API on the user's behalf.
 
+### Proof Key for Code Exchange (PKCE)
+
+> User Authorization Endpoint
+
+```json
+{
+  ...
+  "code_challenge": "671608a33392cee13585063953a86d396dffd15222d83ef958f43a2804ac7fb2",
+  "code_challenge_method": "S256"
+}
+```
+
+> Token Exchange Endpoint
+
+```json
+{
+  ...
+  "code_verifier": "fdsuiafhjbkewbfnmdxzvbuicxlhkvnemwavx"
+}
+```
+
+PKCE proves the app that started the authorization flow is the same app that finishes the flow. You can read more about 
+it [here](https://www.oauth.com/oauth2-servers/pkce/).
+
+Here's what you need to know:
```

Comment:
I really like this focus on the core of what developers need to know without getting lost in the details!